IdentityCore/src/webview/embeddedWebview/ui/ios/MSIDWebviewUIController.m

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import "MSIDWebviewUIController.h"
+#import "UIApplication+MSIDExtensions.h"
+
+@interface MSIDWebviewUIController ( )
+{
+    UIActivityIndicatorView *_loadingIndicator;
+    __weak UIViewController *_parentController;
+    UIModalPresentationStyle _presentationType;
+}
+
+@end
+
+@implementation MSIDWebviewUIController
+
+- (id)initWithContext:(id<MSIDRequestContext>)context
+{
+    self = [super init];
+    if (self)
+    {
+        _context = context;
+    }
+    
+    return self;
+}
+
+- (BOOL)createAndLoadView:(NSError **)error;
+{
+    // Get UI container to hold the webview
+    // Need parent controller to proceed
+    if (![self obtainParentController])
+    {
+        if (error)
+        {
+            *error = MSIDCreateError(MSIDErrorDomain, MSIDErrorNoMainViewController, @"The Application does not have a current ViewController", nil, nil, nil, _context.correlationId, nil);
+        }
+        return NO;
+    }
+    UIView *rootView = [self view];
+    [rootView setFrame:[[UIScreen mainScreen] bounds]];
```

Comment:
just curious, why are we changing the frame of the view?

Reply:
We could either set the frame during init or set it here when we try to load view. I guess we don't want to rely on the default value of frame, therefore we set it ourselves :)